### PR TITLE
Fix Jenkins version to last known stable.

### PR DESCRIPTION
**Pull request by wedaly**
Apparently the latest version of Jenkins has a show-stopping bug: https://issues.jenkins-ci.org/browse/JENKINS-20123

This change ensures that we install the same version we're using in the Jenkins cluster, which does not have this problem.

@jzoldak @feanil

### CODE REVIEW

File: playbooks/roles/jenkins_master/defaults/main.yml [hunk 1]
```diff
@@ -53,4 +56,5 @@ jenkins_debian_pkgs:
     - nginx
     - git
     - maven
+    - daemon
```
Comment (feanil) [inline playbooks/roles/jenkins_master/defaults/main.yml:59]: what does this package do?

**Comment by feanil** [comment]
Instead of grabbing the file, I would continue to use the jenkins apt repo and just lock the correct version
`apt: pkg=jenkins={{jenkins_version}}`  where `jenkins_version` is set to the appropriate version.

**Comment by jzoldak** [comment]
:thumbsup: tested the changes locally by cherry-picking against another branch

File: playbooks/roles/jenkins_master/defaults/main.yml (hunk 1 above)
Comment (wedaly) [inline playbooks/roles/jenkins_master/defaults/main.yml:59]: The Jenkins debian package installs some scripts into /etc/init.d that depend on daemon.

**Comment by wedaly** [comment]
@feanil As far as I can tell, they don't provide old versions in the apt repo.  I'm following the recommendations they provide here for installing older versions:

http://pkg.jenkins-ci.org/debian/

**Comment by feanil** [comment]
Ah I see, looking at the sources list, it does not provide that version, really weird.  It provides 1.528, and 1.536 but nothing in between.

```
1.536 - 
1.528 - 
1.424.6+dfsg-1ubuntu0.2 - 
1.424.6+dfsg-1ubuntu0.1 - 
1.424.6+dfsg-1 - 
```

Change looks good to merge.

**Pull request merged**